CONTRIBUTING.md

```diff
@@ -173,46 +173,6 @@ that are missing in earlier minor versions.
   make check
   ```
 
-### Following our coding conventions
-
-CI pipeline will automatically reject any PRs which do not follow our coding
```

Comment:
I think we should at least keep a link to the styleguide here, otherwise people might not find it themselves.